src/pages/_app.js

```diff
@@ -31,6 +32,10 @@ const G_TAG_KEY = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS
 export default function App({ Component, pageProps }) {
   useGoogleTagsManager(G_TAG_KEY)
 
+  const [applyTabIndex, setApplyTabIndex] = useState(false)
+
+  pageProps = { ...pageProps, applyTabIndex, setApplyTabIndex }
```

Comment:
Please remove. This will mutate page props here and pass them along to All `Component`s in the render.